pyproject.toml

```diff
@@ -26,10 +26,10 @@ rechunker = "bin.rechunker:main"
 
 [tool.poetry.dependencies]
 python = ">=3.8,<3.13"
-jax = "*"
+jax = "<0.8.0"
```

Comment:
Because https://github.com/jax-ml/jax/pull/30386

```
AttributeError: jax.lib.xla_bridge.get_backend is deprecated and will be removed in JAX v0.8.0; use jax.extend.backend.get_backend.
```